### DISCUSSION TITLE
article/e3e6ec8a-bde4-4a71-b661-926212346ae1

### DISCUSSION THREAD

**Discussion by giscus** [siuze/NotionNext#10] [Announcements]
# article/e3e6ec8a-bde4-4a71-b661-926212346ae1

画君年少时, Notion, GitHub, 博客, 叶修齐

https://notion.siuze.top/article/e3e6ec8a-bde4-4a71-b661-926212346ae1

**Comment by Guanchishan** [comment]
野好！
